frontend/src/views/UserDetails/index.vue

```diff
@@ -9,6 +9,10 @@ import UserPermissionHistoryTable from "@/components/UserPermissionHistoryTable"
 import BreadCrumbs from "@/components/UI/BreadCrumbs.vue";
 import type { BreadCrumbType } from "@/types/BreadCrumbTypes";
 import { ManagePermissionsRoute } from "@/router/routes";
+import { useQuery } from "@tanstack/vue-query";
+import { AdminMgmtApiService } from "@/services/ApiServiceFactory";
+import { AdminRoleAuthGroup } from "fam-admin-mgmt-api/model";
```

Comment:
Seems not used.

Reply:
https://github.com/bcgov/nr-forests-access-management/pull/1671/commits/274eccd8df4387a6030070e578ba5b6f86518c56